great_expectations/rule_based_profiler/data_assistant/onboarding_data_assistant.py

```diff
@@ -303,7 +303,7 @@ def _build_numeric_columns_rule() -> Rule:
 
         # Step-1: Instantiate "ColumnDomainBuilder" for selecting numeric columns (but not "ID-type" columns).
 
-        numeric_column_type_domain_builder: DomainBuilder = ColumnDomainBuilder(
+        numeric_column_type_domain_builder = ColumnDomainBuilder(
```

Comment:
Reason why this type was taken away?

Reply:
Oh good question. I had made other modifications in this file that I reverted but removed this one since we don't typically put a type hint when instantiating (since it's pretty redundant). Plus this one is not really correct and leads to autocomplete issues (since ColumnDomainBuilder has different fields).